### ISSUE TITLE
MicaForEveryone Isnt running any more.

### ISSUE THREAD

**Issue by Fenix899** [MicaForEveryone/MicaForEveryone#487]
[x] This is not about other apps being displayed weird or etc.
[x] This is not about rendering issues that caused by a bug in the Windows.
[x] I've read [FAQ](https://github.com/MicaForEveryone/MicaForEveryone/wiki/FAQ).
[x] I've checked other issues and this is not a duplicate.

when i go on the file : MicaForEveryone-x64-Release-Installer
i install it and its not showing in Hidden Apps on desktop i click on it still nothing in Hidden Apps.

1. Go to File Explorer 
2. Click on Downloads 
3. Scroll down to 
MicaForEveryone-x64-Release.zip
4. Install it
5. Go to Hidden Apps

**Expected behavior**
Not seeing micaforeveryone

**Screenshots**
![Screenshot 2024-12-30 145646](https://github.com/user-attachments/assets/71f64b7f-d11c-4299-b1f8-b4b4ac990cc4)
**Windows Version**
Windows 11 Home 24H2
Note: If you know How to fix the issue tell me.

**Comment by C4RP3N0CT3M** [comment]
I'm getting the same behavior. I did manage to get it to open by deleting the app's appdata, but then as soon as I reloaded my appdata.conf it stopped working again.

Edit: The one thing I'm noticing is that no matter how I launch the program, I'm unable to tick the setting to launch as admin at logon. I'm guessing that's the setting that's causing issues.

**Comment by BlankDout** [comment]
Yeah, this is happening to me too. I opened Task Manager to check on it, seems to crash before it displays anything.
